.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 120,
+  "trailingComma": "all",
+  "singleQuote": true
```

Comment:
I don't love this, but I can live with it :)

Reply:
What would you prefer? I just grabbed this off of some blog post about authoring packages.

Reply:
Apologies, I was just joking about my irrational preference for double quotes 😁